src/js/components/LineUpdate/index.ts

```diff
@@ -0,0 +1,61 @@
+import Component from "@/core/component";
+import view from "@/components/LineForm/view";
+import { ILine } from "@/types";
+import { stationStore } from "@/store";
+import { lineService } from "@/service";
+import { createElement } from "@/utils/dom";
+import { $ } from "@/utils/dom";
+
+class LineUpdate extends Component<ILine> {
+  protected initDom(): void {
+    this.$container = createElement({
+      tag: "form",
+      id: "update-line",
+    });
+  }
+
+  protected bindEvents(): void {
+    this.$container.addEventListener("submit", (e: Event) => {
```

Comment:
```suggestion
    const { $container } = this;
    $container.addEventListener("submit", (e: Event) => {
```
전체적으로 $container를 사용중이라서 이렇게 this라도 없앨 수 있도록 해보면 좋을 것 같아요!